hibernate-jpa/build.gradle

```diff
@@ -15,7 +15,8 @@ dependencies {
     api(libs.managed.hibernate.core)
     api(libs.jakarta.transaction.api)
     api(mnValidation.micronaut.validation)
-    api(mnData.micronaut.data.tx.hibernate)
+    compileOnly(mnData.micronaut.data.tx.hibernate)
```

Comment:
I have removed the dependency cycle with Micronaut Data. I think the long term solution would be to move data.tx to the sql module.